pkg/server/claims_test.go

```diff
@@ -87,13 +95,19 @@ func TestTokenClaimString(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			t.Parallel()
 
-			got, err := tokenClaimString(tc.token, tc.value, tc.required)
-			if diff := cmp.Diff(tc.want, got); diff != "" {
-				t.Errorf("mismatch (-want, +got):\n%s", diff)
-			}
-			if msg := testutil.DiffErrString(err, tc.expErrMsg); msg != "" {
-				t.Fatalf(msg)
-			}
+			runCase(t, tc.token, tc.value, tc.required, tc.want, tc.expErrMsg)
 		})
 	}
 }
+
+func runCase[T any](t *testing.T, token jwt.Token, value string, required bool, want T, expErrMsg string) {
```

Comment:
I think this can be re-inlined now that it's only called in one place

Reply:
Re-inlined by instantiating with `any` 🤮